boms/user/server/ee/pom.xml

```diff
@@ -436,7 +436,7 @@
             <id>skip.preview</id>
             <activation>
                 <property>
-                    <name>!quickly</name>
+                    <name>!no.preview.build</name>
```

Comment:
This and the change below result in the preview boms being built when only -Dquickly is used, which I expect would be common(ish).

I think it's ok though; that adds an immaterial amount of time to running such a build.